CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+
+cmake_minimum_required(VERSION 3.15)
+
+project(wasi-libc
+  LANGUAGES ASM C)
+
+option(ENABLE_THREADS "enable threads" OFF)
+
+add_compile_options(-target wasm32-unknown-none)
```

Comment:
For WASI libc, we build for wasm32-wasi.

Reply:
Oops, this is clearly a think-o.